aten/src/ATen/native/native_functions.yaml

```diff
@@ -6709,19 +6709,19 @@
 - func: _to_cpu(Tensor[] tensors) -> Tensor[]
   variants: function
 
-- func: to_dense(Tensor self, ScalarType? dtype=None) -> Tensor
+- func: to_dense(Tensor self, *, ScalarType? dtype=None, bool? masked_grad=None) -> Tensor
```

Comment:
Ah, you can't make dtype kwarg-only. That'll be BC breaking technically and might cause all kinds of headaches. But I think it's ok to make masked_grad kwarg-only.

Reply:
Right. Fixed.